api/src/service/domain/workflow/workflowitem_create.ts

```diff
@@ -201,6 +204,17 @@ export async function createWorkflowitem(
     return new AlreadyExists(ctx, workflowitemCreated, workflowitemCreated.workflowitem.id);
   }
 
+  logger.trace({ item: workflowitemCreated }, "Check if assignee exists");
+  const isUserExists = await repository.userExists(
+    workflowitemCreated.workflowitem.assignee
+  );
+
+  if(!isUserExists) {
```

Comment:
I would also not name the const `isUserExists` - this sounds gramatically incorrect - just `userExists` is fine